### ISSUE TITLE
Unresolvable Dependency error in Laravel Jetstream with Livewire

### ISSUE THREAD

**Issue by Kayrah87** [Jeroen-G/Flickr#15]
Having followed the instructions in your documentation I am now getting a 500 error in Laravel Livewire whilst using the package in a Livewire Component.
Error: `[previous exception] [object] (Illuminate\\Contracts\\Container\\BindingResolutionException(code: 0): Unresolvable dependency resolving [Parameter #0 [ <required> string $apiKey ]] in class JeroenG\\Flickr\\Api at /Users/kay/Work/Repos/Personal/site/vendor/laravel/framework/src/Illuminate/Container/Container.php:1053)`

Component: 
`class Liveside extends Component
{
    public $images;

    public function mount(Flickr $flickr)
    {
        $flickr_arguments = [
          'user_id' => '68669878@N00',
          'extras' => 'url_o',
        ];
        $this->images = $flickr->request('flickr.people.getPhotos', $flickr_arguments);
    }
    public function render()
    {
        return view('livewire.liveside');
    }
}`

I have added the serviceprovider in app.php, as well as the facade. I have also added the array entry into config/services.php

Any suggestions?

**Comment by Jeroen-G** [comment]
What did you add to the app service provider?

**Comment by Kayrah87** [comment]
As per the instructions, the provider was added: `JeroenG\Flickr\FlickrServiceProvider::class,` in the section 'Package Service Providers'.
I didn't add the facade as I'm running Laravel 8.
I have since resolved this issue though. 

The problem was that the key was not getting added to the instantiation because it didn't exist in the env() variables.
I had added it to the .env file, but what was required was to clear the config cache using `php artisan config:clear`

Perhaps worth a note on the docs for future Laravel peeps like me :)
